docs/products/kafka/concepts/configuration-backup.rst

```diff
@@ -0,0 +1,33 @@
+Configuration backups for Apache Kafka®
+=======================================
+
+Aiven for Apache Kafka® includes **configuration backups** that automatically back up key Apache Kafka® service configurations for you without any additional costs. 
+
+Suppose the Apache Kafka® service is powered off/on or an incident causes the Apache Kafka cluster to stop functioning. In that case, the configuration backup enables the restoration of your Apache Kafka® service to its previous state.
+
+.. note:: 
+    The configuration backups feature is currently available for new projects in Aiven.
+
+Whats included in configuration backups
+----------------------------------------
+
+The following data are backed up in the configuration backups:
+
+* Apache Kafka® topic-related configurations.
+* Everything related to Schema Registry (schemas, ids, subjects, compatibility levels).
+* Apache Kafka® Connect configurations.
+
+Benefits of configuration backups
+-------------------------------------
+Some of the key benefits of configuration backups include the following: 
+
+* Configuration backups are automatically enabled and stored in the Cloud storage.
+* Configurations are backed up in 3 hours intervals.
+* It helps with speedy disaster recovery.
```

Comment:
I would add here `speedy disaster recovery in certain situations*` *In a case when all nods are lost and current configurations are not accessible anymore.

When one or two nodes are lost, usually, we do not lose a configuration. Losing all nodes or brain split could lead to configuration lost, but this is quite rare situation. So "quick re-creation of Apache Kafka® services" is a main use-case here.

Reply:
Added the info about the nodes lost as a note.